src/release/rollups.md

```diff
@@ -22,8 +22,19 @@ queue has been merged.
 
 ## Making a Rollup
 
-1. Using the interface on [Homu queue], select a few pull requests and then use
-   "rollup" button to make one. (The text about fairness can be ignored.)
+1. Using the interface on [Homu queue], select pull requests and then
+   use "rollup" button to make a rollup pull request. (The text about
+   fairness can be ignored.)
```

Comment:
Can we remove this text if it's not useful?

Reply:
@jyn514 what part specifically?, you meant all that block?. I've left that first phrase as a quick what needs to be done first sentence, because this is really a procedure and then have added this Important note section to more explicitly try to explain what we need to be careful about.

Reply:
I mean the bit about fairness. It sounds like it's not useful, so we should remove it from bors.rlo

Reply:
I have no idea, but I'd leave that as a followup to make this change in bors.rlo and remove the sentence here.